qemu/tests/cfg/virtio_mem_dynamic_memslots_with_migration.cfg

```diff
@@ -0,0 +1,23 @@
+- virtio_mem_dynamic_memslots_with_migration:
+    only Linux
+    no RHEL.6 RHEL.7 RHEL.8
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    no s390x
+    required_qemu = [8.2.0,)
+    type = virtio_mem_dynamic_memslots_with_migration
+    virt_test_type = qemu
+    login_timeout = 240
```

Comment:
@mcasquer just my question, why do we need it here?

Reply:
In fact is not needed, just waste from other cases, code updated !